app/views/users/show.html.erb

```diff
@@ -1,107 +1,15 @@
-         Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt
-         mollit anim id est laborum.
-
-
-         Lorem ipsum dolor sit amet, consectetur adipiscing elit,
-        sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
-         Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris
-         nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in
-         reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.
-         Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt
-         mollit anim id est laborum.
-
-         Lorem ipsum dolor sit amet, consectetur adipiscing elit,
-        sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
-         Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris
-         nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in
-         reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.
-         Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt
-         mollit anim id est laborum.
-    </p>
-</div>
-
-<div class="posts-list-container">
-    <h4>Post #<%= Random.rand(1..9999) %> </h4>
-    <p class="posts-content">
-        Lorem ipsum dolor sit amet, consectetur adipiscing elit,
-        sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
-         Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris
-         nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in
-         reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.
-         Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt
-         mollit anim id est laborum.
+        <%= @post[0].author.bio %>
     </p>
-    <div class="interaction-container">
-        <span class="comments-counter">Comments X</span>
-        <span class="likes-counter">Likes Y</span>
-    </div>
 </div>
 
-<div class="posts-list-container">
-    <h4>Post #<%= Random.rand(1..9999) %> </h4>
-    <p class="posts-content">
-        Lorem ipsum dolor sit amet, consectetur adipiscing elit,
-        sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
-         Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris
-         nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in
-         reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.
-         Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt
-         mollit anim id est laborum.
-    </p>
-    <div class="interaction-container">
-        <span class="comments-counter">Comments X</span>
-        <span class="likes-counter">Likes Y</span>
-    </div>
+<%= render @post %>
+<div class="btn-container-post">
+<button class="btn-posts">See all Posts</button>
```

Comment:
- [ ] Kindly make this button into a link that navigates to the post index page for the user.